README.md

```diff
@@ -0,0 +1,204 @@
+<a name="#readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#table-of-contents)
+- [📖 my-account ](#my-account)
+  - [🛠 Built With ](#built-with)
+    - [Tech Stack ](#tech-stack)
+    - [Key Features ](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [💻 Getting Started ](#getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#authors)
+  - [🔭 Future Features ](#future-features)
+  - [🤝 Contributing ](#contributing)
+  - [⭐️ Show your support ](#️show-your-support)
+  - [🙏 Acknowledgments ](#acknowledgments)
+  - [❓ FAQ ](#faq)
+    - [`Can we use the application as an organization`](#can-we-use-the-application-as-an-organization)
+    - [`Answer`](#answer)
+  - [`Where can we reach out to you ?`](#where-can-we-reach-out-to-you)
+    - [`Answer`](#answer-1)
+  - [📝 License ](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [my-account](#my-account) <a name="about-project"></a>
+
+The my-account application is a fully functional website that allow you to manage your expenses. The website was created with Ruby on Rails. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The following technologies have been used for the project development.
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://expressjs.com/">Rails.js</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+The key features of the application are:
+
+- Create, Read and Delete expenses and categories
+- Authentication with Devise
+- Authorization rules for users
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live demo <a name="live-demo"></a>
+
+Please watch the presentation video at this [link](https://www.loom.com/share/678be0ed9a374054bc2f6ba576425e89)
```

Comment:
- [ ] [OPTIONAL]
I want to acknowledge your hard work, @diarisdiakite  I have a couple of suggestions to enhance your presentation:

1. Including the mobile view of the presentation would have been exceptionally beneficial, especially since the project requirement emphasizes mobile responsiveness. The menu icon, Login, and some other elements appear a little disorganized because they were primarily styled for mobile view.

2. While it's commendable to communicate the effort invested in developing the app to your employer/client/reviewer, it might be more professional not to explicitly mention the presence of bugs. Instead, you could convey that the app is still a work in progress, leaving room for further refinement.